source/webapp/sites/all/modules/custom/checkbook_solr/config/smart_search_config.json

```diff
@@ -218,6 +213,22 @@
           "250-500": "[250,499.99]",
           "<250": "[1,249.99]"
         }
+      },
+      "check_amount": {
+        "title": "Check Amount",
+        "intervals": {
+          ">100M": "[100000000,*]",
+          "51M-100M": "[50000000,100000000]",
+          "26M-50M": "[25000000,50000000]",
+          "11M-25M": "[10000000,25000000]",
+          "1M-10M": "[1000000,10000000]",
+          "<1M": "[*,999999.99]"
```

Comment:
please change to 99,999,999.99 , otherwise value 100M will appear in both ranges

Reply:
Replicated the range, what we have transactions pages. I fixed overlapping issue on smart search now.